packages/selenium/src/WebDriverInjectorExtensions.cs

```diff
@@ -8,38 +9,28 @@ namespace Deque.AxeCore.Selenium
 {
     internal static class WebDriverInjectorExtensions
     {
+        private static readonly JsonSerializerSettings JsonSerializerSettings = new JsonSerializerSettings
+        {
+            Formatting = Formatting.None,
+            NullValueHandling = NullValueHandling.Include
+        };
+
         /// <summary>
-        /// Injects Axe script into frames.
+        ///  Yeilds contexts such that every yield will be in a different frame context.
```

Comment:
I feel like this is verging on "too clever" territory - I don't think it's very obvious from usage that enumerating through the returned object will be implicitly changing the state of the contained webdriver and that the actual value returned by the enumerator is meaningless, especially for a future maintainer with only passing familiarity with C# (which I think is a likely scenario for this sort of integration library).

I think it would be clearer to instead write this (and its inner overload) along the lines of:

```csharp
internal static void ForEachFrameContext(this IWebDriver driver, Action callback)
```